package.json

```
@@ -92,5 +108,13 @@
         "name": "Emu_E2E"
       }
     }
+  },
+  "@react-native-community/bob": {
+    "source": "src",
+    "output": "lib",
+    "targets": [
+      "commonjs",
```

Since you're using Flow, I'd suggest to use the `copyFlow` option as well so that flow definitions are copied

```suggestion
      ["commonjs", {"copyFlow": true}],
```

---

Makes sense, thanks :pray: